Error occurred: 'gbk' codec can't encode character '\U0001f621' in position 0: illegal multibyte sequence
Traceback (most recent call last):
  File "C:\Users\seven\Desktop\qiAgent\to_mofa\agent-hub\qiAgent\qiagent\pdfprocessor.py", line 20, in extract_text_from_pdf
    with open(pdf_path, 'rb') as file:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '../resume/resume.pdf'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\seven\Desktop\qiAgent\.venv\Lib\site-packages\mofa\agent_build\base\base_agent.py", line 260, in wrapper
    func(*args, **kwargs)
  File "C:\Users\seven\Desktop\qiAgent\to_mofa\agent-hub\qiAgent\qiagent\main.py", line 57, in run
    text = work(file_path)
           ^^^^^^^^^^^^^^^
  File "C:\Users\seven\Desktop\qiAgent\to_mofa\agent-hub\qiAgent\qiagent\main.py", line 18, in work
    pdf_content = pdf_processor.extract_text_from_pdf(pdf_path)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\seven\Desktop\qiAgent\to_mofa\agent-hub\qiAgent\qiagent\pdfprocessor.py", line 32, in extract_text_from_pdf
    print(f"\U0001f621PDF�ı���ȡʧ��: {e}")
UnicodeEncodeError: 'gbk' codec can't encode character '\U0001f621' in position 0: illegal multibyte sequence
Traceback (most recent call last):
